### ISSUE TITLE
Index lost

### ISSUE THREAD

**Issue by afedulov** [jethrokuan/z#100]
I ran out of space on my harddrive, had to clean up and reboot and the search index is lost. I would like to debug what has happened and consider backing z's data up. Where is the index file located? I

**Comment by jethrokuan** [comment]
```
echo $Z_DATA
```

should tell you where it is.

**Comment by afedulov** [comment]
Thanks @jethrokuan. The file got truncated. Any idea why this could happen when the disk get's full?

**Comment by markstos** [comment]
@afedulov Yes. The OS wrote half of the file, ran out of disk space and couldn't finish.

I think this support issue can be closed.
